build/x86_64_debian/Dockerfile

```diff
@@ -10,8 +10,7 @@ RUN apt-get update -y && apt-get install -y \
         fakeroot \
         curl \
   && gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
-  && curl -sSL https://get.rvm.io | /bin/bash -s stable \
+  && (curl -sSL https://get.rvm.io | /bin/bash -s stable) \
```

Comment:
Let's move this after the `git config` commands, for consistency.
In general, let's make sure the commands are executed in the same order in all 3 `Dockerfile`s, or that there's a comment explaining why the order is different.

Reply:
Done